### PR TITLE
A rub share links

### PR DESCRIPTION
**Pull request by AndriyRubanskui**

### CODE REVIEW

File: client/src/components/Share/index.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.scss';
+
+const Share = () => (
+        <div className='services'>
+          <span>Our Social Networks</span>
+          <div>
+          <a href="https://www.facebook.com/" target='blank' className='facebook-icon'></a>
```
Comment (anna-kuzii) [inline client/src/components/Share/index.jsx]: are u sure? not _blank?

File: client/src/components/Share/index.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.scss';
+
+const Share = () => (
```
Comment (anna-kuzii) [inline client/src/components/Share/index.jsx]: can we export here?

File: client/src/components/Share/style.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
+@import '../styles/variables.scss';
+
+.services {
+  display: flex;
+  justify-content: space-between;
+  width: 220px;
+
+  & span {
```
Comment (anna-kuzii) [inline client/src/components/Share/style.scss]: do we need & here?

File: client/src/components/Share/style.scss [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  display: block;
+  width: 20px;
+  height: 20px;
+  background: url('../../assets/img/twitter-normal.svg') no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.linkedin-icon {
+  display: block;
```
Comment (anna-kuzii) [inline client/src/components/Share/style.scss]: we can combine all of them, no?

File: client/src/components/Share/style.scss (hunk 3 above)
Comment (AndriyRubanskui) [reply client/src/components/Share/style.scss]: no

File: client/src/components/Share/index.jsx (hunk 2 above)
Comment (AndriyRubanskui) [reply client/src/components/Share/index.jsx]: Yes, we can)

File: client/src/components/Share/index.jsx (hunk 1 above)
Comment (AndriyRubanskui) [reply client/src/components/Share/index.jsx]: ohhh yeee)

File: client/src/components/Share/style.scss (hunk 4 above)
Comment (AndriyRubanskui) [reply client/src/components/Share/style.scss]: of course)

**Pull request merged**